### PR TITLE
no exit code as counter

### PR DESCRIPTION
**Pull request by DMyachin**
Added flag `--no-exit-counter` that allows get exit code 0 if no any errors appears.

### CODE REVIEW

**Comment by DMyachin** [comment]
https://github.com/Exodus-Privacy/exodus-standalone/issues/34

File: exodus_analyze.py [hunk 1]
```diff
@@ -105,6 +107,13 @@ def main():
         default=None,
         help='comma-separated ids of trackers to ignore'
     )
+    parser.add_argument(
+        '--no-exit-counter',
```
Comment (pnu-s) [inline exodus_analyze.py:111]: Could it be possible to have a shorter option name? I find `--no-exit-counter` quite long, especially with 2 dashes in the option name.

Comment (pnu-s) [inline exodus_analyze.py:111]: Out of curiosity, why preferring this implementation rather than the --exit-code option which was in the original issue?

**Review by pnu-s — COMMENTED** [review]
Thanks for your PR!

Tested it locally and it works as expected.

I made a couple of comments/questions

File: exodus_analyze.py (hunk 1 above)
Comment (DMyachin) [reply exodus_analyze.py]: I use '-e'. Is it OK?

Comment (DMyachin) [reply exodus_analyze.py]: ¯\\_(ツ)_/¯ 

` --exit-code` in new commit

**Comment by DMyachin** [comment]
@pnu-s ?

**Comment by pnu-s** [comment]
Sorry for the delayed answer! 

My proposal was not just to use the `exit-code` name rather than `no-exit-code` but to offer the user to specify its error code (0 to avoid failing totally, 1 to overwrite the current behavior which returns the number of trackers, or other)

That would improve the user experience in my opinion

In terms of code, it would mean:
* if no trackers, always return 0
* if trackers, either return the specified exit code if existing otherwise the number of trackers as a RC

**Comment by DMyachin** [comment]
See new commit

```python
    if all([counter, forced_code is not None]):
        sys.exit(forced_code)
    sys.exit(counter)
```
0 is False in Python.

If counter is not zero (has trackers) AND code overwriting is set to any int value, the value will be used else counter.

**Comment by pnu-s** [comment]
LGTM, thanks for your quick changes!

You'll just need to remove that extra line to make the linter pass, and I'll be able to merge it :rocket:

**Review by pnu-s — APPROVED** [review]

**Pull request merged**